src/main/java/appeng/core/transformer/AE2ELTransformer.java

```diff
@@ -188,4 +193,39 @@ public String map(final String name) {
 
     }
 
+    private static class SafeClassWriter extends ClassWriter {
+
+        public SafeClassWriter(int flags) {
+            super(flags);
+        }
+
+        @Override
+        protected String getCommonSuperClass(final String type1, final String type2) {
+            Class<?> c, d;
+            // clueless
+            ClassLoader classLoader = MinecraftServer.class.getClassLoader();
```

Comment:
Better to just use `Launch.classLoader` here, `MinecraftServer.class` may have not been loaded yet.

Reply:
yeah